ddp-workspace/projects/ddp-dsm-ui/src/app/STORE/selectors/Utils/participantsList.util.ts

```diff
@@ -0,0 +1,19 @@
+import {ParticipantModel} from '../../models/participant.model';
+
+export const generateParticipantsList = (participants: ParticipantModel[], settings: any): Array<any> => {
+  const actDefs = settings?.activityDefinitions;
+
+  return participants instanceof Array && actDefs ? participants.map(pt => ({
+    ID: pt.esData.profile.hruid,
```

Comment:
In the Store, we also have a utils folder, that will help is for calculations that might be needed for selectors which are processing data.